Sources/Classes/core/Downloader/MiniAppClient.swift

```diff
@@ -264,10 +267,23 @@ internal class MiniAppClient: NSObject, URLSessionDownloadDelegate {
             delegate?.downloadFileTaskCompleted(url: "", error: NSError.downloadingFailed())
             return
         }
-        checkFileSignature(destinationURL: destinationURL, location: location)
+        fileDownloadingCompleted(destinationURL: destinationURL, at: location)
+    }
+
+    private func fileDownloadingCompleted(destinationURL: String, at location: URL) {
```

Comment:
maybe `fileDownloadingCompleted` should be renamed `fileDownloadCompleted`, but I'm not sure. Also I think we got the `destinationURL` wrong from the beginning, it should be `originURLString`, isn't it?

Reply:
Let me check

Reply:
That's right. It's a bit confusing. Renamed it now.